### PR TITLE
fix(region): check user credential length when exec task

### PR DESCRIPTION
**Pull request by ioito**
**What this PR does / why we need it**:

- [x] Smoke testing completed

If yes, write the release branches name in the below format and submit the related cherry-pick PR:
- release/3.11

### CODE REVIEW

File: pkg/cloudcommon/db/taskman/tasks.go [hunk 1]
```diff
@@ -321,6 +321,15 @@ func (manager *STaskManager) NewTask(
 		ParentTaskId: parentTaskId,
 	}
 
+	if field, ok := reflect.TypeOf(task).Elem().FieldByName("user_cred"); ok {
```
Comment (wanyaoqi) [inline pkg/cloudcommon/db/taskman/tasks.go:324]: 这个 width 值感觉可以在 init 函数中初始化，可以不用每次做反射

Comment (ioito) [reply pkg/cloudcommon/db/taskman/tasks.go]: 已更新

**Pull request merged**